### DISCUSSION TITLE
Issue: Computed properties can access on the front end by Livewire.find('componentId').set('computedProperty', 'new Value')

### DISCUSSION THREAD

**Discussion by firer777** [livewire/livewire#8387] [7. Bugs (archived)]
Computed properties can access on the front end by Livewire.find('componentId').set('computedProperty', 'new Value');
I added #[Locked] to Computed property but not working;
If computed property value is a eloquent model and using more inside component and change computed property value by js to string, will error in blade example: $this->user->email and error when using $this->user->fill([...])->update();

Please fix and Lock computed property :(

**Comment by danie-ramdhani** [comment]
Locked attribute only can be used on properties (https://livewire.laravel.com/docs/locked)

If you want your function cannot be accessed in frontend, use private or protected modifier.

Function with computed attribute can be accessed in frontend with any modifier (public, protected, private) see https://github.com/livewire/livewire/discussions/7823

**Reply by firer777** [reply]
yeb, now issue computed property can access by frontend and i want add locked to computed property for deny frontend access but not working
#[Computed]
#[Locked]
protected function demo() {...}

**Comment by RealMrHex** [comment]
any solution?
